oxy_fitting_ver2.py

```diff
@@ -0,0 +1,447 @@
+
+# coding: utf-8
+
+# # THIS IS ANALYSIS OF 1707 (PO6E) based upon NOAA's oxy fitting
+# 
+# ## Density Matching Instead of Interpolation
+
+# In[1]:
+
+
+import sys
+sys.path.append('/Users/k3jackson/p06e/ctd_proc')
+sys.path.append('/Users/k3jackson/p06e/Kennycode/')
+import matplotlib
+matplotlib.use('TkAgg')
+import matplotlib.pyplot as plt
+import scipy
+import numpy as np
+import libODF_process_ctd as process_ctd
+import libODF_sbe_reader as sbe_rd
+import libODF_sbe_equations_dict as sbe_eq
+import gsw
+import pandas as pd
+
+
+
```

Comment:
At some point these configs will be put into a config file, once we can work out where it should be.